met-api/tests/unit/api/test_engagement.py

```diff
@@ -64,6 +64,23 @@ def test_get_engagements(client, jwt, session, engagement_info):  # pylint:disab
     assert created_eng.get('content') == rv.json.get('content')
 
 
+@pytest.mark.parametrize('engagement_info', [TestEngagementInfo.engagement1])
+def test_get_engagements_paginated_advanced_search(client, jwt,
```

Comment:
may be change the method name to emphasize the purpose
i will call it test_search_engagements_by_status or something like that..

Reply:
updated, thanks